### ISSUE TITLE
decide to apply street_car as HCT or not in accessibility

### ISSUE THREAD

**Issue by bricegnichols** [psrc/soundcast#624]
Street car has been moved to mode "b" (local) for assignment, but the transit_stops.csv file should have the stops labeled as street_car. Should we update the accessibility script to consider street_car stops as HCT when computing buffered variables?

https://github.com/psrc/soundcast/blob/asim_develop/scripts/accessibility/accessibility.py#L344-L345

**Comment by stefancoe** [comment]
@bricegnichols the main purpose of this file is to create parcel to transit stop distances by mode for the Daysim pathtype models. Since we don't have a streetcar submode, we need to make sure the streetcar stops are lumped in with the right category during the accessibility script.

**Comment by bricegnichols** [comment]
We updated transit_stops.csv to consolidate transit stops into a single row with an indicator for whether different submodes use the stop. For instance, a stop that serves buses and street_car will have 1 in each of those columns. These are referred to as post_rtp inputs for 2023
